docs/kpod-stats.1.md

```diff
@@ -28,7 +32,43 @@ Pretty-print images using a Go template
 
 ## EXAMPLE
 
-TODO
+```
+# kpod -a --no-stream
```

Comment:
is this right and line 49?  You don't need the 'stats' keyword?  Also each one of these has a blank line and then a # command prompt on a second line after each example.  I'd delete lines 45, 53 and 69.